SDK.markdown

```diff
@@ -5,3 +5,203 @@ permalink: /sdk/
 ---
```

Comment:
Let's add `sidebar: true` to SDK as well.

Reply:
nice, missed this one